libr/debug/debug.c

```diff
@@ -1561,7 +1561,17 @@ static int show_syscall(RDebug *dbg, const char *sysreg) {
 	return reg;
 }
 
-// continue execution until a syscall is found, then return its syscall number or -1 on error
+/* Continue execution until a syscall is found.
+ *
+ * If no plugin is selected, read the x86 sn register in order to determine the syscall. See https://www2.lauterbach.com/pdf/debugger_x86.pdf.
+ * In case of bug, it is recommended to use a debugger plugin such as -d "windbg://file.exe" on Windows.
+ *
+ * first argument, RDebug: debugger configuration structure.
```

Comment:
rdebug is the instance that holds the state of rdebug, it have nothing to do with the debugger configuration